storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -699,7 +761,7 @@ bool xbcloud_put(Object_store *store, const std::string &container,
   }
   auto thread = h.run();
 
-  while (!has_errors) {
+  do {
```

Comment:
why this change? and how it affects other storages?

Reply:
It is just to avoid compilation warning. 
../storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc:849:7: warning: variable 'res' may be uninitialized when used here [-Wconditional-uninitialized]
  if (res == XB_STREAM_READ_ERROR || has_errors ||
 
 Else logic remains the same